ocaml/idl/datamodel.ml

```diff
@@ -4220,6 +4240,68 @@ let alert =
     ()
 *)
 
+(** network sriov **)
+module Network_sriov = struct
+  let lifecycle = [Published, rel_kolkata, ""]
+
+  let sriov_configuration_mode = Enum ("sriov_configuration_mode",
+    [
+      "sysfs", "Configure network sriov by sysfs, do not need reboot";
```

Comment:
Grammar:
* ..., does not require reboot
* ..., requires reboot

I think it is strange to have an _unknown mode_.

Please use SRIOV consistently in the documentation. There are SR-IOV,  SRIOV, sriov in use already.